charts/prometheus/README.md

```diff
@@ -2,57 +2,82 @@
 
 [Prometheus](https://prometheus.io/), a [Cloud Native Computing Foundation](https://cncf.io/) project, is a systems and service monitoring system. It collects metrics from configured targets at given intervals, evaluates rule expressions, displays the results, and can trigger alerts if some condition is observed to be true.
 
-## TL;DR;
-
-```console
-helm install stable/prometheus
-```
-
-## Introduction
-
 This chart bootstraps a [Prometheus](https://prometheus.io/) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
 
 ## Prerequisites
 
 - Kubernetes 1.3+ with Beta APIs enabled
 
-## Installing the Chart
+## Get Repo Info
 
-To install the chart with the release name `my-release`:
+```console
+$ helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+$ helm repo add stable https://kubernetes-charts.storage.googleapis.com/
+$ helm repo update
+```
+
+_See [helm repo](https://helm.sh/docs/helm/helm_repo/) for command documentation._
+
+## Install Chart
 
 ```console
-helm install --name my-release stable/prometheus
+# Helm 3
+$ helm install [RELEASE_NAME] prometheus-community/prometheus
+
+# Helm 2
+$ helm install --name [RELEASE_NAME] prometheus-community/prometheus
 ```
 
-The command deploys Prometheus on the Kubernetes cluster in the default configuration. The [configuration](#configuration) section lists the parameters that can be configured during installation.
+_See [configuration](#configuration) below._
+
+_See [helm install](https://helm.sh/docs/helm/helm_install/) for command documentation._
 
-> **Tip**: List all releases using `helm list`
+## Dependencies
 
-## Uninstalling the Chart
+By default this chart installs additional, dependent charts:
 
-To uninstall/delete the `my-release` deployment:
+- [stable/kube-state-metrics](https://github.com/helm/charts/tree/master/stable/kube-state-metrics)
+
+To disable the dependency during installation, set `kubeStateMetrics.enabled` to `false`.
+
+_See [helm dependency](https://helm.sh/docs/helm/helm_dependency/) for command documentation._
+
+## Uninstall Chart
 
 ```console
-helm delete my-release
+# Helm 3
+$ helm uninstall [RELEASE_NAME]
```

Comment:
Why not just `helm delete` for both Helm versions? In Helm v3 `helm del` actually purges the chart by default, right?

Reply:
For Helm 2, `--purge` is recommended. In Helm 3 `delete` is an alias for `uninstall`, but has no `--purge` flag. Therefore it can't be identical.

Reply:
Follow-up from chat: `delete` was added as an alias to ease some CI workflows, but it's not encouraged for Helm 3 generally, which is why it's not in the docs. The Helm `uninstall` concept itself is different now with Helm 3 architectural changes than default Helm 2 `delete` , but in the end maps to Helm 2's helm `delete --purge`.

Mainly we are trying to help lead users to Helm 3, since there is only now 3 months until Helm 2 is unsupported.

To be clear, `delete` can be used with Helm 3, but not encouraged due to user confusion between the different Helm versions, when `--purge` is needed or not for a full uninstall, leading to different results depending on version. It's a lot to explain fully so IMO easiest to document here as different commands for Helm 3 and 2, then just remove Helm 2 references after Helm 2 support ends.

You already said in Slack that makes sense, but just so others who may be interested can catch up as well 😄